crates/asm-spec/asm.yml

```diff
@@ -347,54 +347,11 @@ StateRead:
               stack_in: [slot_ix]
               stack_out: [len]
 
-            State:
-              opcode: 0x39
-              description: |
-                Access a range of words from the state value located in the slot at `slot_ix`.
-
-                Push `len` words from the value onto the stack, starting from the word at `value_ix`.  
-
-                The argument delta `false` accesses pre state.
-                The argument delta `true` accesses post state.
-              panics:
-                - slot_ix is out of range.
-                - The range `value_ix..(value_ix + len)` is out of range.
-                - delta is not a boolean.
-              stack_in: [slot_ix, value_ix, len, delta]
-              stack_out:
-                elem: word
-                len: len 
-
-            StateLen:
-              opcode: 0x3A
-              short: SLEN
-              description: |
-                Get the length of a state value at a specified `slot_ix`.
-
-                The argument delta `false` accesses pre state.
-                The argument delta `true` accesses post state.
-
-                Returns the length of the state value.
-              panics:
-                - slot_ix is out of range.
-                - delta is not a boolean.
-              stack_in: [slot_ix, delta]
-              stack_out: [len]
-
-            NumSlots:
+            DecisionVarSlots:
               opcode: 0x3B
```

Comment:
I think this and the following byte numbers need to be adjusted. We should have a test for this somehow

Reply:
Good spot!